### PR TITLE
zulip-notifications 0.1.1

### PR DESCRIPTION
**Pull request by sethmurphy18**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

I would like to make 2 notes about my step:
- I **do** use external libraries; however, I have included them in my repository in an offline cache, in the event that downloading them from npm fails (which knowing npm is entirely possible).
- I **do** technically speaking `cd` twice; however, both times I do so is in a subshell, so the working directory of the main Bitrise process **_shouldn't_** change, unless my understanding of how `bash` works is horribly wrong (which is possible as I use `zsh` mainly).

### CODE REVIEW

**Comment by sethmurphy18** [comment]
Well, something failed, but the link is dead, so I can't see what's wrong. Oh well, guess I'll wait until Monday to find out.

File: steps/zulip-notifications/0.1.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+      \ Valid:  \"Stream1, some-user@someOrg.zulipchat.com, Stream2\"\n  Valid:  \"Stream1,some-user@someOrg.zulipchat.com,Stream2\"\n
+      \ Valid:  \"Stream1 , some-suer@someOrg.zulipchat.com , Stream2\"\n```\n"
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: List of recipients to send build notifications to
+    title: Zulip Recipeients
+  zulip_step_recipients: null
+- opts:
+    content: '{buildEmoji} Build {buildNumber} has {buildResult}!'
```
Comment (lpusok) [inline steps/zulip-notifications/0.1.1/step.yml]: The build failed when using previous cli version, due to unknown key: `error: not supported key found in options: content`
(https://github.com/bitrise-io/bitrise/releases/download/1.24.0/bitrise-Darwin-x86_64)

File: steps/zulip-notifications/0.1.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+      \ Valid:  \"Stream1, some-user@someOrg.zulipchat.com, Stream2\"\n  Valid:  \"Stream1,some-user@someOrg.zulipchat.com,Stream2\"\n
+      \ Valid:  \"Stream1 , some-suer@someOrg.zulipchat.com , Stream2\"\n```\n"
+    is_expand: true
+    is_required: false
+    is_sensitive: true
+    summary: List of recipients to send pull request build notifications to
+    title: Zulip Recipients (Pull Request)
+  zulip_step_recipients_pull_request: null
+- opts:
+    content: |-
```
Comment (lpusok) [inline steps/zulip-notifications/0.1.1/step.yml]: Similar as above, unknown key.

File: steps/zulip-notifications/0.1.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+description: |
+  Send build notifications to Zulip based on build status and release status.
+website: https://github.com/diolapp/bitrise-step-zulip-notifications
+source_code_url: https://github.com/diolapp/bitrise-step-zulip-notifications
+support_url: https://github.com/diolapp/bitrise-step-zulip-notifications/issues
+published_at: 2019-04-06T21:35:08.670357-04:00
+source:
+  git: https://github.com/diolapp/bitrise-step-zulip-notifications.git
+  commit: da5f6fc8624152cb160b92b95798f673a4810da1
+host_os_tags:
```
Comment (lpusok) [inline steps/zulip-notifications/0.1.1/step.yml]: Can remove host_os_tags section, as it is not used by the cli.

File: steps/zulip-notifications/0.1.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
```
Comment (lpusok) [inline steps/zulip-notifications/0.1.1/step.yml]: Can remove toolkit section, as step.sh is used by default.

**Comment by lpusok** [comment]
Commented the cause in the previous comments.

File: steps/zulip-notifications/0.1.1/step.yml (hunk 1 above)
Comment (sethmurphy18) [reply steps/zulip-notifications/0.1.1/step.yml]: So the content tag is not supported on older versions of Bitrise? How would I enable a multiline input with preserved newlines then, like the script step?

Comment (lszucs) [reply steps/zulip-notifications/0.1.1/step.yml]: You can do it just like the script step: https://github.com/bitrise-io/bitrise-steplib/blob/master/steps/script/1.1.5/step.yml#L42-L65 :)

**Comment by lpusok** [comment]
Closing PR, as changes and re-share is needed. Sorry for the not instantly obvious CI failure, and still happy to accept a fixed step.

**Pull request closed without merging**